static/app/components/core/text/text.tsx

```diff
@@ -178,7 +172,7 @@ export const Heading = styled(
   display: ${p => (p.ellipsis ? 'inline-block' : undefined)};
 
   font-family: ${p => (p.monospace ? p.theme.text.familyMono : p.theme.text.family)};
-  font-weight: ${p => ((p.bold ?? true) ? p.theme.fontWeight.bold : undefined)};
+  font-weight: bold;
```

Comment:
This should still use the theme value! Our `bold` is technically `600` rather than `700` 😅 

```suggestion
  font-weight: ${p => p.theme.fontWeight.bold};
```

Reply:
Aghhh, good catch!